homeassistant/strings.json

```diff
@@ -45,7 +45,8 @@
         "latitude": "Latitude",
         "location": "Location",
         "pin": "PIN Code",
-        "mode": "Mode"
+        "mode": "Mode",
+        "metering_point": "Metering point"
```

Comment:
I don't think you are allowed to change this, especially not without an approved [architecture](https://github.com/home-assistant/architecture) issue.

Reply:
I will remove this line because it is not required to be present in this file in the next commit. Thanks.

Reply:
After removing this line from the file, hassfest is throwing an error from the pipeline. However, when I keep above line in the file, It works. Can you please help to fix this issue?